<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Editor</class>
 <widget class="QMainWindow" name="Editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>611</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Editor</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab_description">
       <property name="toolTip">
        <string notr="true"/>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <attribute name="title">
        <string>Beskrivning</string>
       </attribute>
       <attribute name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Rummets namn och beskrivning&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Namn</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="text">
           <string>Hus</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Beskrivning</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit">
          <property name="statusTip">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_item">
       <property name="toolTip">
        <string/>
       </property>
       <attribute name="title">
        <string>Objekt</string>
       </attribute>
       <attribute name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Objekt&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Namn</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QCheckBox" name="checkBox_3">
          <property name="text">
           <string>Går att sälja</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="text">
           <string>Kaffekopp</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Beskrivning</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QCheckBox" name="checkBox_4">
          <property name="text">
           <string>Går att kasta</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QWidget" name="widget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Värde</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QTextEdit" name="textEdit_2">
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Droid Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-size:9pt;&quot;&gt;En riktigt fin kaffekopp&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QCheckBox" name="checkBox_2">
          <property name="text">
           <string>Går att plocka upp</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_npcs">
       <attribute name="title">
        <string>Personer</string>
       </attribute>
       <attribute name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Person&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QWidget" name="widget_namn" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_name">
             <property name="minimumSize">
              <size>
               <width>10</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Namn</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="edit_name">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_description" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_description">
             <property name="text">
              <string>Beskrivning</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="edit_description"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_dialogs" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label_dialog">
             <property name="text">
              <string>Dialog</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="edit_dialog"/>
           </item>
           <item>
            <widget class="QLabel" name="label_recieved_dialog">
             <property name="text">
              <string>Dialog vid mottaget objekt</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="edit_recieved_dialog"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_modifier" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_hair">
             <property name="text">
              <string>Hårfärg</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="edit_hair">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="label_height">
             <property name="text">
              <string>Längd</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_width">
             <property name="text">
              <string>Bredd</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="edit_wants">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_weight">
             <property name="text">
              <string>Vikt</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_wants">
             <property name="text">
              <string>Begär</string>
             </property>
            </widget>
           </item>
           <item row="4" column="3">
            <widget class="QCheckBox" name="checkbox_Merchant">
             <property name="text">
              <string>Försäljare</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="edit_width"/>
           </item>
           <item row="2" column="3">
            <widget class="QSpinBox" name="edit_height"/>
           </item>
           <item row="0" column="3">
            <widget class="QSpinBox" name="edit_weight"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_exits">
       <attribute name="title">
        <string>Utgångar</string>
       </attribute>
       <attribute name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ändra rummets utgångar&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_N">
          <property name="text">
           <string>Nord</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="combo_N">
          <item>
           <property name="text">
            <string>Inget</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Rum 1: Hus</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Rum 2: Fult hus</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="combo_S">
          <item>
           <property name="text">
            <string>Inget</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Rum 1: Hus</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Rum 2: Fult hus</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="combo_V">
          <item>
           <property name="text">
            <string>Inget</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Rum 1: Hus</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Rum 2: Fult hus</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="combo_E">
          <item>
           <property name="text">
            <string>Inget</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Rum 1: Hus</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Rum 2: Fult hus</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_S">
          <property name="text">
           <string>Syd</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_W">
          <property name="text">
           <string>Väst</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_E">
          <property name="text">
           <string>Öst</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="right_widget" native="true">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_room">
         <property name="text">
          <string>Rum</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combobox_rooms">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Rum 1: Hus</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Rum 2: Fult hus</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttom_new_room">
         <property name="text">
          <string>Lägg till nytt rum</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_items">
         <property name="text">
          <string>Objekt i rummet</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="list_items">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Kaffekopp</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Låda</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Pannkaka</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>John</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_item">
         <property name="text">
          <string>Lägg till nytt objekt</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_npcs">
         <property name="text">
          <string>Personer i rummet</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="list_npcs">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_npc">
         <property name="text">
          <string>Lägg till ny person</string>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>list_npcs</zorder>
      <zorder>label_room</zorder>
      <zorder>combobox_rooms</zorder>
      <zorder>buttom_new_room</zorder>
      <zorder>label_items</zorder>
      <zorder>list_items</zorder>
      <zorder>button_item</zorder>
      <zorder>label_npcs</zorder>
      <zorder>button_npc</zorder>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>648</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArkiv">
    <property name="title">
     <string>Arkiv</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuRedigera">
    <property name="title">
     <string>Redigera</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuVisa">
    <property name="title">
     <string>Visa</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuInstallning">
    <property name="title">
     <string>Inställningar</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHjalp">
    <property name="title">
     <string>Hjälp</string>
    </property>
   </widget>
   <addaction name="menuArkiv"/>
   <addaction name="menuRedigera"/>
   <addaction name="menuVisa"/>
   <addaction name="menuInstallning"/>
   <addaction name="menuHjalp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
